common/main/en_AU.xml

```diff
@@ -6711,12 +6711,6 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 				<symbol>SHP</symbol>
 				<symbol alt="narrow">↑↑↑</symbol>
 			</currency>
-			<currency type="SLL">
```

Comment:
@srl295 - Just out of curiosity, why are we removing instead of changing to inheritance?

Reply:
Well, the result is the same... and we cannot say that en_AU vetters would vote to always inherit, what we do know is they voted for an explicit value that happened to be the same (at the time) as the inherited value, _instead_ of voting for inheritance.

Reply:
Thank you for the explanation!